Sources/Testing/Traits/ConditionTrait+Macro.swift

```diff
@@ -92,9 +92,16 @@ extension Trait where Self == ConditionTrait {
     _ condition: @escaping @Sendable () -> Bool
   ) -> Self {
     // TODO: Semantic capture of platform name/version (rather than just a comment)
-    Self(
+    let message: Comment = if let message {
+      message
+    } else if let version {
+      "Obsolete as of \(_description(ofPlatformName: platformName, version: version))"
```

Comment:
I know this is the existing behavior, but I wonder if this text would align better with the "obsolete**d**" term in source:
```suggestion
      "Obsoleted in \(_description(ofPlatformName: platformName, version: version))"
```

Reply:
`"Obsoleted as of macOS 99.0"`

vs.

`"Obsoleted in macOS 99.0"`

Reply:
I don't feel strongly, but as it's old code I'm going to leave it as-is for now. We can open another PR if you like.

Reply:
It's really "Obsolete" vs. "Obsolete**d**" that I was focused on. The latter would more closely match the language keyword which is spelled `obsoleted`.